train.py

```diff
@@ -129,7 +129,16 @@ def collect_features(self, *args):
             text, speaker_id = args
         else:
             text = args[0]
+        global _frontend # _frontend None bug fix
+        if _frontend == None:
+            _frontend = getattr(frontend, hparams.frontend)
         seq = _frontend.text_to_sequence(text, p=hparams.replace_pronunciation_prob)
+        _frontend = None # memory leaking prevention
```

Comment:
I think this is too tricky and I don't  want to do this on LInux, I'm ok on windows though. Could you add a branch to make sure this only happens on windows? e.g.,

```py
if platform.system() == "Windows":
    ....
```